api/v1alpha1/platformoperator_types.go

```diff
@@ -27,8 +27,14 @@ type PlatformOperatorSpec struct {
 
 // PlatformOperatorStatus defines the observed state of PlatformOperator
 type PlatformOperatorStatus struct {
-	InstalledBundleName         string `json:"installedBundleName"`
-	InstalledBundleInstanceName string `json:"installedBundleInstanceName"`
+	InstalledBundle         InstalledBundle `json:"installedPackage"`
+	InstalledBundleInstance string          `json:"installedBundleInstance"`
```

Comment:
I'm not sure either of these fields are needed at this point. I think my expectation is that we'll generated a BundleInstance where the metdata.Name matches the parent PlatformOperator's metadata.Name. And I don't think we need to track an installed Bundle field because that information is already present in the BI's status once the plain provisioner starts reconciling that resource.